content/1.7/back-office/shipping/carriers/Add-edit-carrier.md

```diff
@@ -0,0 +1,202 @@
+ - Maximum package height (cm) an input field, has a tooltip: "Maximum height managed by this carrier. Set the value to "0", or leave this field blank to ignore. The value must be an integer."
+ - Maximum package depth (cm) an input field, has a tooltip: "Maximum height managed by this carrier. Set the value to "0", or leave this field blank to ignore. The value must be an integer."
+ - Maximum package weight (Kg) an input field, has a tooltip: "Maximum height managed by this carrier. Set the value to "0", or leave this field blank to ignore."
+ - Group access, displays the ID and group name in a table, all of them have a checkmark on the left, by default Vistor, guest and customer has checkmarks checked.
+A tooltip is displayed: "Mark the groups that are allowed access to this carrier."
+     - Group name, if this field id checkmarked, all the fields get checkmarked or checkmark gets removed.
+     - Visitor
+     - Guest
+     - Customer
+
+### Call to action
+
+  - **Previous.** If there is an error, this button is disabled. After clicking on it, the previous step is displayed.
+ - **Next.** If there is an error, this button is disabled. After clicking on it, the next step is displayed.
+ - **Finish.** If there is an error, this button is disabled. After clicking on it, the carrier is saved and the [carriers listing]({{<ref "carriers-listing.md">}} "Carriers listing") page is displayed with the following success message: "Successful creation" if the carrier was added or "Successful update" if the carrier was edited.
+
+# Summary (5th step)
+
+### Call to action
+ 
+A call to action button at the top right is displayed:
+
+ - Cancel:
+Pressing the button redirects to the [carriers listing]({{<ref "carriers-listing.md">}} "Carriers listing") page.
+
+## Columns
+
+Firstly on the left the logo is displayed. Whena  logo is uploaded, a trash can icon appears on the right of the logo. When trying to delete the logo, message appears: "Are you sure you want to delete the logo?". Pressing "Ok" will remove the uploaded logo, and the default van logo will re-appear in the same place.
+
+On the right side, from left to the right, the steps of adding/editing a carrier are displayed: 
+
+ - **1 General settings**
+ - **2 Multistore (If multistore is enabled)**
+ - **3 Shipping locations and costs**
+ - **4 Size, weight, and group access**
+ - **5 Summary**
+
+### Carrier summary
+
+Information about the carrier being created is displayed:
+
+![CarrierSummary](/img/CarrierSummary.png)
+
+The carrier's price, and transit time, how the shipping costs are handled, what is the weight range of the delivery that the carrier can make, how the carrier reacts if the weight is higher, what are the delivery zones, and for which customers the carrier is displayed.
+
+Below there is a toggle switch "Enabled". It can be selected as "yes" or "no". Has a tooltip: "Enable the carrier in the front office."
```

Comment:
```suggestion
Below, a toggle "Enabled" is displayed. It can be enabled or disabled. By default, it's enabled. The following tooltip is displayed: "Enable the carrier in the front office."
```